.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -15,7 +15,7 @@ Fixes / closes #????
 - [ ] Added / updated test-suite
 <!-- If this is a feature pull request / breaks compatibility: -->
 <!-- (Otherwise, remove these lines.) -->
-- [ ] Corresponding documentation was added / updated (including any warning and error messages added / removed / modified).
+- [ ] Corresponding documentation was added / updated (including any warning and error messages added / removed / modified, and including reflecting any changes to the grammar in `common.edit_mlg` and running `make doc_gram_rsts`).
```

Comment:
```suggestion
- [ ] If you modified Coq's grammar, run `make doc_gram_rsts` to update the syntax in the documentation (see `docgram/README.md`).
- [ ] Corresponding documentation was added / updated, including any changed warning and error messages.
```

I suggest splitting this--checklist items should kept short so they're easy to read.  I started with Théo suggested wording.

In practice, the doc reviewers will need to pay attention to the docgram changes.  It's not foolproof (fools generally being quite clever...).